includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -504,7 +504,7 @@ public function get_header_logo_id() {
 	 */
 	public function get_header_logo_height() {
 		if ( !empty( $this->settings['header_logo_height'] ) ) {
-			return apply_filters( 'wpo_wcpdf_header_logo_height', $this->settings['header_logo_height'], $this );
+			return apply_filters( 'wpo_wcpdf_header_logo_height', sanitize_text_field($this->settings['header_logo_height']), $this );
```

Comment:
I would apply the `str_replace` that you applied in the earlier commit here (in case someone puts a space between the number and the unit)